JumpscaleLibs/sal/zosv2/ZOSv2.py

```diff
@@ -133,6 +81,17 @@ def reservation_register(self, reservation, expiration_date, identity=None, expi
         return resp.id
 
     def reservation_result(self, reservation_id):
+        return reservation_get(reservation_id).results
+
+    def reservation_get(self, reservation_id):
+        """
+        fetch a specific reservation from BCDB
+        
+        :param reservation_id: reservation ID
+        :type reservation_id: int
+        :return: reservation object
+        :rtype: "tfgrid.workloads.reservation.1
+        """
         return self._actor_workloads.workload_manager.reservation_get(reservation_id).results
```

Comment:
I think this should return the object now instead of the `results` field?

Reply:
right